rgb20/src/schema.rs

```diff
@@ -665,252 +673,7 @@ mod test {
         assert_eq!(id.to_string(), SCHEMA_ID_BECH32);
         assert_eq!(
             id.to_string(),
-            "sch1zcdeayj9vpv852tx2sjzy7esyy82a6nk0gs854ktam24zxee42rqyzg95g"
-        );
-    }
-
-    #[test]
-    fn schema_structure() {
```

Comment:
As test it makes not a lot of sense since we already test the whole schema data with bech32 strings. As for the parsing, we will have YAML and Contractum representations of the schema, which will require their own tests (not ready yet) matching the contractum/YAML version with this schema.